auto-pilot972623843/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I don't know why is this got included in the pr

Reply:
remove it